Gemfile

```diff
@@ -22,4 +22,6 @@ group :development do
   gem 'listen', '>= 3.0.5', '< 3.2'
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'cucumber-rails', require: false
```

Comment:
These two new gems should be under `development` and `test`

Reply:
done!